akka-http/src/main/scala/akka/http/scaladsl/server/ExceptionHandler.scala

```diff
@@ -51,6 +51,11 @@ object ExceptionHandler {
         ctx.request.discardEntityBytes(ctx.materializer)
         ctx.complete((status, info.format(settings.verboseErrorMessages)))
       }
+      case e: EntityStreamSizeException ⇒ ctx ⇒ {
+        ctx.log.error(e, ErrorMessageTemplate, e, RequestEntityTooLarge)
+        ctx.request.discardEntityBytes(ctx.materializer)
+        ctx.complete((RequestEntityTooLarge, e.toString.format(settings.verboseErrorMessages)))
```

Comment:
I think `e.getMessage` would be neater here compared to `e.toString`.

The `.format()` looks a bit like the `e.info.format()` below, but is in fact very different: this is https://www.scala-lang.org/api/current/scala/collection/immutable/StringOps.html#format(args:Any*):String which is not needed here since the exception message doesn't have a placeholder that needs to be filled.

```suggestion
        ctx.complete((RequestEntityTooLarge, e.getMessage))
```

Reply:
I agree. Just pushed this change. Thanks!